internal/helm/service.go

```diff
@@ -537,7 +540,7 @@ func (s service) UpgradeRelease(ctx context.Context, organizationID uint, cluste
 	return nil
 }
 
-func (s service) ListCharts(ctx context.Context, organizationID uint, filter ChartFilter, options Options) (charts ChartList, err error) {
+func (s service) ListCharts(ctx context.Context, organizationID uint, filter ChartFilter, _ Options) (charts ChartList, err error) {
```

Comment:
Not sure where these changes come from but doesn't seem like it's necessary

Reply:
this is linter a linter fix